main.js

```diff
@@ -0,0 +1,315 @@
+// 1
+
+function getGreeting(firstName, lastName) {
+    return(`Hi, "` + firstName + ` ` + lastName + `". What's up?`);
+}
+
+const greeting = getGreeting('John', 'Smith');
+console.log(greeting);
+
+// 2
+
+const realName = 'Wojciech';
+
+function isThisMyName (name) {
+    if (name === realName ) {
+        return true;
+    } else {
+        return false;
+    }
+}
+
+console.log(isThisMyName('Adam'));
+console.log(isThisMyName('Wojciech'));
+
+// 3
+
+function isThisBoolean (value) {
+    return typeof value === 'boolean';
+}
+
+console.log(isThisBoolean(1));
+console.log(isThisBoolean(true));
+
+// 4
+
+const quarterOf = (month) => {
+    switch (true) {
+        case (month <= 3):
+            return 1;
+        case (month <= 6):
+            return 2;
+        case (month <= 9):
+            return 3;
+        default:
+            return 4;
+    }
+}
+
+// 5 - Dawne pierwsze 4
+
+function getCircleArea (radius) {
+    return Math.PI * (radius * radius);
+}
+
+const radius = 5;
+const area = getCircleArea(radius);
+console.log(area);
+
+// 6
+
+function getCirclesAreaSum(radiusOne, radiusTwo) {
+    return getCircleArea(radiusOne) + getCircleArea(radiusTwo);
+}
+
+const circlesAreaSum = getCirclesAreaSum(5, 10);
+console.log(circlesAreaSum);
+
+// 7
+
+function convertMinutesToHours (minutesAmount) {
```

Comment:
There is an unnecessary space here

Reply:
`minutes` are countable
https://dictionary.cambridge.org/grammar/british-grammar/amount-of-number-of-or-quantity-of